libs/consent/editor/src/lib/consentEditView.tsx

```diff
@@ -0,0 +1,135 @@
+import {ApolloError} from '@apollo/client'
+import {stripTypename} from '@wepublish/editor/api'
+import {
+  ConsentValue,
+  MutationUpdateConsentArgs,
+  useConsentQuery,
+  useUpdateConsentMutation
+} from '@wepublish/editor/api-v2'
+import {useMemo, useState} from 'react'
+import {useTranslation} from 'react-i18next'
+import {useNavigate, useParams} from 'react-router-dom'
+import {Form, Message, Schema, toaster} from 'rsuite'
+
+import {ModelTitle} from '@wepublish/ui/editor'
+import {ConsentForm} from './consentForm'
+import {getApiClientV2} from '../apiClientv2'
+
+const mapApiDataToInput = (consent: any): MutationUpdateConsentArgs['consent'] => ({
+  ...stripTypename(consent),
+  name: consent.name,
+  slug: consent.slug,
+  defaultValue: consent.defaultValue
+})
+
+const onErrorToast = (error: ApolloError, slug?: string) => {
+  if (error.message.includes('Unique constraint')) {
+    toaster.push(
+      <Message type="error" showIcon closable duration={3000}>
+        {`A consent with slug '${slug}' already exists. Please choose a different slug.`}
```

Comment:
I'm guessing the error message isn't helpful here? Can we use translations for this?